CMakeLists.txt

```diff
@@ -69,7 +69,7 @@ if(MSVC)
 else()
   option(BUILD_SHARED_LIBS "Build shared libraries" ON)
 endif()
-option(KIDO_BUILD_OSGKIDO "Build osgKido library" ON)
+option(KIDO_BUILD_OSGKIDO "Build osgKido library" ON) # TODO(JS): we probably should remove this option or add another build options for other extensions as well for consistency.
```

Comment:
Re: Packing. The ideal way to package KIDO as a Debian would be to create one `kido` source package that `Build-Depend`s on the build dependencies *all* components. The source package would build a separate binary package for each component (e.g. `kido-optimizer-nlopt`) that `Depend`s only on the runtime dependencies for the headers/library that it contains.

I doubt it is possible to change this structure within one Debian release, since it changes the names (and number) of binary packages built from the source package.

Is this the plan? Or will we stick with a single, monolithic `kido` package for now?